specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Dataset.json

```diff
@@ -3712,8 +3712,8 @@
       }
     },
     "LakeHouseTableDataset": {
-      "x-ms-discriminator-value": "LakeHouseTable",
-      "description": "Microsoft Fabric LakeHouse Table.",
+      "x-ms-discriminator-value": "LakehouseTable",
```

Comment:
@JeffreyRichter @mikeharder I feel like this is an api layer breaking here, but it's not getting detected correctly. because if customers want to pass an object for `LakeHouseTableDataset`, they will have to change the discriminator property type's value from "LakeHouseTable" to "LakehouseTable". Should we enhance our tooling so that we don't miss cases like this? 

but as @JaylenZ8 confirmed offline, since service backend is backward compatible for this, and it can still recognize "LakeHouseTable" I guess it's probably okay for this one?

Reply:
@qiaozha: Please open an issue at https://github.com/Azure/openapi-diff/issues to suggest a bug fix or feature request for the breaking changes detector.

Reply:
Thanks, issue created here https://github.com/Azure/openapi-diff/issues/350